thirdParty/cmake-modules/FindmallocMC.cmake

```diff
@@ -22,6 +22,8 @@
 #   mallocMC_INCLUDE_DIRS    - Include directories for the mallocMC headers.
 #   mallocMC_FOUND           - TRUE if FindmallocMC found a working install
 #   mallocMC_VERSION         - Version in format Major.Minor.Patch
+# and for individual COMPONENTS:
+#   mallocMC_<COMPONENT>_FOUND  - TRUE if FindmallocMC found that component
```

Comment:
Could we add a note which case the resulting ```<COMPONENT>``` part of the string will have? Is it original case or uppercase or will both be defined?